pkg/cache/memcached.go

```diff
@@ -24,6 +25,10 @@ func (cfg *MemcachedConfig) RegisterFlagsWithPrefix(prefix, description string,
 	f.DurationVar(&cfg.Expiration, prefix+"memcached.expiration", 0, description+"How long keys stay in the memcache.")
 }
 
+var cacheBufAllocator = &allocator{
+	pool: pool.New("cache", 0, 100, 40*1024),
```

Comment:
In `prealloc.go' I see that the number of buckets and their sizes is configurable. Do we want the same here? What's the rationale behind the 40KB?

Reply:
> Do we want the same here?

Sure. Not a bad idea.

> What's the rationale behind the 40KB?

Untuned. I just chose 100 buckets and wanted the cap to be 4MB to catch all value sizes in memcached.